rpmbuild/copr_rpmbuild/helpers.py

```diff
@@ -407,10 +407,18 @@ def __init__(self, path):
 
     def __getattr__(self, name):
         value = getattr(self.spec, name)
-        if name == "epoch" and value is not None:
-            return int(value)
         return value
 
+    @property
+    def epoch(self):
+        """
+        Evaluated %{epoch} as integer or None
+        """
+        value = self.safe_attr("epoch")
```

Comment:
This doesn't solve the "multiple" tag value behind %if/else condition, or does it?  Shouldn't we use the `spec.tags(spec.parsed_sections.package).content.epoch.expanded_value` variant?

Reply:
IMHO this should be exactly it :-)


`self.safe_attr` returns `getattr(self.tags, name).expanded_value` while `self.tags` are `self.spec.tags(self.spec.parsed_sections.package).content`

So basically:

```python
getattr(self.spec.tags(self.spec.parsed_sections.package).content, name).expanded_value
```

Reply:
Nice then, I think. Thank you for the explanation. @nforro would you mind having a quick look?

Reply:
LGTM :+1: